### ISSUE TITLE
Question about the paper

### ISSUE THREAD

**Issue by Bartender-bit** [TripleJoy/SAM2MOT#3]
"Assuming object B is occluded, we first compare the logits scores of objects A and B. If object A’s score is significantly higher, it suggests object B is occluded. If their logits scores are similar, we need to distinguish between two possibilities: object A’s score has gradually decreased due to lack of reinitialization, or object B’s score has suddenly dropped due to occlusion. To clarify this, we compute the variance of their logits scores over the past N frames, as shown in Equation 4. Typically, object A will show higher variance because its score decline is gradual, while object B’s drop is abrupt. By analyzing this variance, we can identify object B as the occluded object."

In the paper (section 3.3), you mentioned that the variance of A would be larger. However, if "B’s drop is abrupt. ", shouldn't B's variance be larger instead? Am I misunderstanding something here?

By the way, I'd like to ask if you plan to open-source the training code in the future?

Thanks!

**Comment by TripleJoy** [comment]
> "Assuming object B is occluded, we first compare the logits scores of objects A and B. If object A’s score is significantly higher, it suggests object B is occluded. If their logits scores are similar, we need to distinguish between two possibilities: object A’s score has gradually decreased due to lack of reinitialization, or object B’s score has suddenly dropped due to occlusion. To clarify this, we compute the variance of their logits scores over the past N frames, as shown in Equation 4. Typically, object A will show higher variance because its score decline is gradual, while object B’s drop is abrupt. By analyzing this variance, we can identify object B as the occluded object."
> 
> In the paper (section 3.3), you mentioned that the variance of A would be larger. However, if "B’s drop is abrupt. ", shouldn't B's variance be larger instead? Am I misunderstanding something here?
> 
> By the way, I'd like to ask if you plan to open-source the training code in the future?
> 
> Thanks!
Apologies, in Section 3.3, as you pointed out, the variance of A should be smaller than that of B. The code logic was designed with the assumption that A has a smaller variance, but we made an error in the paper. We will revise it accordingly. Additionally, we plan to open-source the code, so stay tuned.

**Comment by Bartender-bit** [comment]
OK, I thought at first that only the inference code would be open source. It would be great if the training code could be open sourced as you said. Thanks！

**Comment by nikoliazekter** [comment]
@Bartender-bit the authors use pre-trained detectors and a pre-trained SAM2.1 model. There will be no training code because there is nothing to train.

**Comment by Bartender-bit** [comment]
oh ok got it now,thanks
